source/release-notes/changelogs/backend/changelog-backend-2022.rst

```diff
@@ -1,3 +1,10 @@
+.. _backend_20220615:
+
+15 June 2022 Release
+~~~~~~~~~~~~~~~~~~~~
+
+- Introduced ability to configure field-level permissions with :ref:`Data API <data-api>`.
```

Comment:
```suggestion
- Introduced ability to configure :ref:`field-level permissions with the Data API <field-level-permissions>`.
```
Adding this ref, which will break currently, but should land in https://github.com/mongodb/docs-app-services/pull/57/files#diff-164464af6d913aa338132b5dafd1b94251f65c1d101cf12937c7070d1582c6f2R407 before release.